keck_vnc_launcher.py

```diff
@@ -91,6 +94,10 @@ def create_parser():
     ## add options
     parser.add_argument("-c", "--config", dest="config", type=str,
         help="Path to local configuration file.")
+    parser.add_argument("-vncserver", type=str,
+        help="Name of VNC server to connect to.  Takes precedence over all.")
+    parser.add_argument( '-vncports', nargs='+', type=str,
```

Comment:
I suggest double dashes to follow the conventions already in use, so `--vncserver` and `--vncports`

Reply:
whoops, just an oversight.  will fix.